packages/expo-av/src/Video.types.ts

```diff
@@ -119,6 +119,10 @@ export type VideoProps = {
    * An optional property to pass custom styles to the poster image.
    */
   posterStyle?: ImageProps['style'];
+  /**
+   * An optional property to pass custom styles to the <Video /> component.
```

Comment:
```suggestion
   * An optional property to pass custom styles to the internal video component.
```

This form might confuse users, since they are importing `Video` component form `av`, and this prop refers to internal video component.

Ideally we probably should rename `style` to `containerStyle` and use `style` for the internal video component instead of `videoStyle` but this could result in regressions and UI breaks for the package users so I think that it's reasonable to leave it as-is, at least for now. 😉

Reply:
agree with you, we try not to make destructive changes.